tests/Application/app/AccountCollection.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Collection;
+
+class AccountCollection extends Collection
```

Comment:
The Collection class is marked as generic in the [stubs](https://github.com/nunomaduro/larastan/blob/master/stubs/EloquentCollection.stub). So any child class should also mark it as generic. You can do it like this
```php
/**
 * @template TModel
 * @extends Collection<TModel>
 */
class AccountCollection extends Collection
{}
```